### DISCUSSION TITLE
Contributors List Still Showing users who do not have commits in the Repository.

### DISCUSSION THREAD

**Discussion by upsteer** [community/community#40605] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body
This is in Repo Page.
Firstly, abc owns the repo, they gave me the permission to contribute. xyz is the one who was invited to contribute. But I accidentally committed to the repo with `mno` user email. Eventhough I have re-written the whole history, the contributor list still shows the old emails, there is just one branch, deleted the branch which `mno` had committed. Git Log also does not show any commits.

**Comment by ossanna16** [comment]
Hi there @upsteer and welcome to our community! Thank you for the feedback!

**Reply by RokerHRO** [reply]
…I ran into the same problem. Still no solution here?
